### ISSUE TITLE
Changing the default `PGPORT` settings breaks the flow

### ISSUE THREAD

**Issue by zstadler** [openmaptiles/openmaptiles#1354]
Currently it is impossible to use the OMT flow if the value of `PGPORT` is modified in `.env`.

## To reproduce:
1. Modify `.env` to use `PGPORT=54321`
2. Run `make start-db`

```
$ make start-db
Starting postgres docker compose target using default image (no recreate if exists)
Creating master_postgres_1 ... done
Wait for PostgreSQL to start...
docker-compose run --rm --user=1000:1000 openmaptiles-tools pgwait
Creating master_openmaptiles-tools_run ... done
... gave up waiting for Postgres:   PGHOST=postgres PGDATABASE=openmaptiles PGUSER=openmaptiles PGPORT=54321 pg_isready
ERROR: 1
make: *** [Makefile:306: start-db] Error 1
```

## Analysis
Port `5432` is always used by the `openmaptiles/postgress` container. As a result, all other containers need to access the PGDB using the `5432` port.

## Alternative solutions:
1. Remove the `PGPORT` from the `.env` file and replace all other occurrences of `PGPORT` with hard-code `5432` in the `openmaptiles` and `openmaptiles-tools` repositories.
2. Modify `openmaptiles/postgress` Docker image to consider `PGPORT` and start the DB using `postgress -p ${PGPORT:-5432}`

**Comment by TomPohys** [comment]
Thanks for pointing this out. I would like to keep the possibility to change the PORT, so PR for alternative 2 is very welcome.

**Comment by zstadler** [comment]
@TomPohys,

To address this issue on top of https://github.com/openmaptiles/openmaptiles-tools/pull/403, something like this need to be done in this repository:

```
services:
  postgres:
    ports:
      - "${PGPORT:-5432}:${PGPORT:-5432}"
```

How would you suggest this change be done given that the `main` branch is using version `6.1` of the postgis docker image rather than `latest`?

**Comment by ondrejlohnisky** [comment]
I know this is not related. But I've got the same error while all config was correct. And even stoping and removing the container did not help.

I had to remove all docker images and containers. At least those that relate.
`docker stop $(docker ps –filter status=running -q)`
`docker rm $(docker ps –filter status=exited -q)`

`docker rmi $(docker images -q)`

I could not find any help on this so i hope it helps someone.
